src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
- *
+ * @version TBD
```

Comment:
Do we have tests for this template? If not, can we add tests?

Reply:
I can add a snapshot test for this template in particular. But it'll be only loaded for twentyseventeen, given that we're hooking on top of their filter, to make the read more link look like what they want on the design: https://github.com/WordPress/twentyseventeen/blob/master/functions.php#L222

Reply:
We should be adding snapshots for every template we have, regardless of whether it's used in one theme or all the themes we support.

Reply:
We should be adding tests for all templates for sure.